### PR TITLE
Mesh 1092/treasury compliance manager rpc

### PR DESCRIPTION
**Pull request by vkomenda**

### CODE REVIEW

File: pallets/compliance-manager/src/lib.rs [hunk 1]
```diff
@@ -199,8 +199,8 @@ impl From<AssetTransferRules> for AssetTransferRulesResult {
             is_paused: asset_rules.is_paused,
             rules: asset_rules
                 .rules
-                .iter()
-                .map(|rule| AssetTransferRuleResult::from(rule.clone()))
+                .into_iter()
+                .map(AssetTransferRuleResult::from)
```
Comment (fmiguelgarcia) [inline pallets/compliance-manager/src/lib.rs:203]: :heart:

File: rpc/Cargo.toml [hunk 2]
```diff
@@ -7,29 +7,38 @@ edition = "2018"
...
 node-rpc-runtime-api = { path = "./runtime-api" }
 
-polymesh-primitives = { path = "../primitives", default-features = false }
+pallet-compliance-manager = { path = "../pallets/compliance-manager", default-features = false }
 pallet-pips = { path = "../pallets/pips", default-features = false }
+polymesh-primitives = { path = "../primitives", default-features = false }
 
 # Substrate
 codec = { package = "parity-scale-codec", version = "1.2.0" }
+frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, rev = 'c771821cae2dcb5a8808a19fae8122c0b9ae8499'}
```
Comment (fmiguelgarcia) [inline rpc/Cargo.toml:16]: Rev is wrong on several lines. It should use `tag = "v2.0.0-rc4" `

File: rpc/runtime-api/Cargo.toml [hunk 3]
```diff
@@ -6,31 +6,35 @@ edition = "2018"
...
+pallet-compliance-manager = { path = "../../pallets/compliance-manager", default-features = false }
 pallet-pips = { path = "../../pallets/pips", default-features = false  }
+polymesh-primitives = { path = "../../primitives", default-features = false }
 
 # Other
 serde = { version = "1.0.104", optional = true, features = ["derive"] }
 
 # Substrate
 codec = { package = "parity-scale-codec", version = "1.2.0", default-features = false, features = ["derive"] }
+frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, rev = 'c771821cae2dcb5a8808a19fae8122c0b9ae8499'}
```
Comment (fmiguelgarcia) [inline rpc/runtime-api/Cargo.toml:18]: ditto

File: rpc/Cargo.toml (hunk 2 above)
Comment (vkomenda) [reply rpc/Cargo.toml:16]: Well, it's the least of my worries just looking at the list of merge conflicts after yesterday's merges.

**Pull request merged**